Userland/Applications/Magnifier/MagnifierWidget.cpp

```diff
@@ -24,6 +25,12 @@ void MagnifierWidget::set_scale_factor(int scale_factor)
     update();
 }
 
+void MagnifierWidget::show_grid(bool new_value)
+{
+    m_show_grid = new_value;
+    update();
```

Comment:
You can early-return here if `m_show_grid == new_value`.

Reply:
I could, but it would literally never happen, so why add more lines of code?